anticipy/forecast_plot.py

```diff
@@ -334,12 +333,13 @@ def plot_forecast(df_fcast, path, output='html', width=None,
         | - date (timestamp)
         | - model (str) : ID for the forecast model
         | - y (float) : Value of the time series in that sample
-        | - is_actuals (bool) : True for actuals samples, False for forecasted samples # noqa
+        | - is_actuals (bool) : True for actuals samples, False for
+        |                       forecasted samples # noqa
```

Comment:
you can remove `#noqa` here, it's for ignoring PEP8 long line warning

Reply:
I think we should raise a different issue to fix any docstrings that need fixing.